pkg/sys/smart/smart_test.go

```diff
@@ -16,6 +14,8 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+//go:build !skip
```

Comment:
Move all build constraints to beginning of the file.

Reply:
build constraints at the top leads to goheader lint failure.

Reply:
Weird I have never seen this style anywhere. You would need to fix it accordingly.

Reply:
done

Reply:
build constraints are not the topmost header - https://github.com/minio/kes/blob/master/internal/fips/fips.go#L5 check how to add them.

It was an older Go code that had this and some packages in runtime library are still in the old format. Backward compatibility is still maintained.

But they moved to build tags right before the `package <name>`